### PR TITLE
Add streamlined CardTip component

### PR DESCRIPTION
**Pull request by judastheo**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Integration tests added
- [x] Visual regressions screenshots are up to date
- [x] Design validated

### CODE REVIEW

File: packages/sandbox/e2e/android/screenshots/CardTip.png
Comment (kimak) [inline packages/sandbox/e2e/android/screenshots/CardTip.png:1]: @ianrobertlambert this one to review on Android

File: packages/sandbox/e2e/ios/screenshots/CardTip.png
Comment (kimak) [inline packages/sandbox/e2e/ios/screenshots/CardTip.png:1]: @ianrobertlambert here on iOS

File: packages/sandbox/e2e/android/screenshots/CardTip.png
Comment (ianrobertlambert) [reply packages/sandbox/e2e/android/screenshots/CardTip.png:1]: All good from my side, thanks! 🙌
I've marked as "Done" on the board in Notion. ✅

File: packages/sandbox/e2e/ios/screenshots/CardTip.png
Comment (ianrobertlambert) [reply packages/sandbox/e2e/ios/screenshots/CardTip.png:1]: All good from my side, thanks! 😊
I've marked as "Done" on the board. ✅

**Review by kimak — APPROVED** [review]

**Pull request merged**